pkgs/games/anki/default.nix

```diff
@@ -38,25 +46,25 @@ in buildPythonApplication rec {
         # "http://ankisrs.net/download/mirror/${name}.tgz"
         # "http://ankisrs.net/download/mirror/archive/${name}.tgz"
       ];
-      sha256 = "0yjyxgpk79rplz9z2r93kmlk09ari6xxfrz1cfm2yl9v8zfw1n6l";
+      sha256 = "56eef02545160388770ec4c923354b485269bcca666a8def7403216555a26117";
     };
 
-    propagatedBuildInputs = [ pyqt4 sqlalchemy pyaudio beautifulsoup httplib2 send2trash ]
+    propagatedBuildInputs = [ pyqt sqlalchemy
+      beautifulsoup4 send2trash pyaudio requests decorator markdown ]
                             ++ lib.optional plotsSupport matplotlib;
 
     checkInputs = [ pytest glibcLocales nose ];
 
     buildInputs = [ lame mplayer libpulseaudio  ];
 
+    makeWrapperArgs = [
+        ''--prefix PATH ':' "${lame}/bin:${mplayer}/bin"''
+        ''--prefix QT_PLUGIN_PATH : "${qt5.qtbase.bin}/lib/qt-5.${lib.versions.minor qt5.qtbase.version}/plugins"''
```

Comment:
Usually this should come from the profile. I don't know what happens if it loads some plugins from here and then the rest from the profile.

Reply:
I set the plugin path, because Anki required older version of Qt, but it is no longer needed if I update to version 2.1.6-beta1.